docs(stack): reorganize navigation into evaluate and deployment paths (#1605)
Consensys/doc.linea: docs/stack/features/fast-finality.mdx

---
title: Finality design
description: >-
  Application-level and settlement finality choices for Lineth deployments
sidebar_position: 3
image: /img/socialCards/instant-finality.jpg
---

> Use this page to compare application-level finality and settlement finality when designing a Lineth (formerly the Linea Stack) deployment. Timing and irreversibility depend on validator topology, consensus assumptions, proof generation and submission, and the selected finalization layer.

Reviewer: ```suggestion
This page describes application-level and settlement finality in a Lineth (formerly the Linea Stack) deployment. Timing and irreversibility depend on validator topology, consensus assumptions, proof generation and submission, and the selected finalization layer.
```
